spec/acceptance/suites/default/10_rsyslog_integration_spec.rb

```diff
@@ -232,39 +232,65 @@ def restart_rsyslog(hosts)
       end
     end
 
-    it 'should generate a local yum log' do
-      hosts.each do |host|
+    hosts.each do |host|
+      # DNF systems don't log to yum.log
+      next if host.which('dnf')
+
+      it 'should generate a local yum log' do
```

Comment:
Any reason we can't add a test here against `dnf.rpm.log` which is very similar to `yum.log` of previous releases?

Reply:
While we could do the local check, it would be for naught as I did some digging and it looks like `dnf` will only log it's own log files, not to syslog.